### PR TITLE
Use OpenJDK10

### PR DESCRIPTION
**Pull request by devinbileck**
Oracle JDK10 is EOL and is no longer available for download. So switch to OpenJDK10 in Gradle builds.

### CODE REVIEW

**Comment by ManfredKarrer** [comment]
Great thanks for working on that. I will wait for more review from @cbeams and @ripcurlx ...

File: .travis.yml
Comment (cbeams) [inline .travis.yml:2]: Glad to see this works without the `install-jdk.sh` bit.

File: common/build.gradle
Comment (cbeams) [inline common/build.gradle:16]: Add the following line here, and delete the `buildscript` block above:

    id 'com.google.osdetector' version '1.6.0'

(Per https://github.com/google/osdetector-gradle-plugin#for-gradle-21-and-higher)

File: common/build.gradle
Comment (cbeams) [inline common/build.gradle]: Why?

File: common/build.gradle
Comment (cbeams) [inline common/build.gradle:19]: Remove along with the `buildscript` block.

File: common/build.gradle
Comment (cbeams) [inline common/build.gradle]: This isn't the name of a class. Assuming you remove the `application` plugin as suggested above, this should be removed too.

File: common/build.gradle
Comment (cbeams) [inline common/build.gradle]: Everything resolvable from Maven Central should also be resolvable from JCenter. Did that not prove true for you? Please remove `mavenCentral()` references here and everywhere else if possible.

**Review by cbeams — CHANGES_REQUESTED** [review]
NACK per inline comments; all fairly minor stuff. Thanks for putting this together, @devinbileck.

File: common/build.gradle
Comment (devinbileck) [reply common/build.gradle:16]: I had to do it this way because I ran into the issue mentioned here: https://github.com/google/osdetector-gradle-plugin/issues/15

File: common/build.gradle
Comment (devinbileck) [reply common/build.gradle]: Whoops. That shouldn't have been added.

File: common/build.gradle
Comment (cbeams) [reply common/build.gradle:16]: Ok, that's a shame. Would be worth a comment about this in the `buildscript` block; otherwise it just looks like it got added due to ignorance of how `plugins {}` works.

Comment (cbeams) [reply common/build.gradle:16]: And you should be able to use `jcenter()` in the buildscript block vs `mavenCentral()`.

**Review by cbeams — APPROVED** [review]

**Comment by cbeams** [comment]
LGTM from my side now, and I've approved the changes.

**Comment by ripcurlx** [comment]
Thanks for the PR @devinbileck! I’ll test the client and release build on Monday if we need to do some additional changes because of the jar with modules config, but I guess our current workaround might fix this problem already.

**Comment by cyphersphinx86** [comment]
Nice footwork @devinbileck
I'm still a little puzzled on how the refered javafx version 11 seamlessly work with jdk version 10, but cool!!!

**Comment by devinbileck** [comment]
> Nice footwork @devinbileck

Thanks! :)

> I'm still a little puzzled on how the refered javafx version 11 seamlessly work with jdk version 10, but cool!!!

As per https://github.com/javafxports/openjdk-jfx/blob/jfx-11/doc-files/release-notes-11.md#release-notes-for-javafx-11:
> JavaFX 11 requires either JDK 10 (which must be an OpenJDK build) or JDK 11.

**Comment by ripcurlx** [comment]
ACK - I tested a release build and everything seems to work. I pushed the verification checksum of the new libraries to this branch. As GitHub seems to be having issues right now I'll wait until [my commit](https://github.com/devinbileck/bisq/commit/c231187daa5a828198268836a2ee68163b5d1d56) will also show up here in the PR. @devinbileck Just one question: Why was it necessary to add the module-path to the configuration as we don't load libraries as java 9 modules within the app?

**Review by ripcurlx — APPROVED** [review]

**Comment by devinbileck** [comment]
> Why was it necessary to add the module-path to the configuration as we don't load libraries as java 9 modules within the app?

@ripcurlx Good point, that may not be necessary. I was following the example from https://openjfx.io/openjfx-docs/#gradle and left that in. Shall I remove it?

**Comment by ManfredKarrer** [comment]
@devinbileck Just tested the changes with installed openjdk. All works fine! Thanks a lot for your help!!!!

**Comment by ripcurlx** [comment]
@devinbileck I'll do some checks tomorrow also try to get the checksum verification working again. Thanks for all your work!

**Pull request merged**
